src/components/molecules/NHSNotifyAuthenticator/NHSNotifyAuthenticator.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-require-imports,unicorn/prefer-module */
+
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react';
+import { usePathname, redirect } from 'next/navigation';
+import { ReactNode, FC } from 'react';
+import { Amplify } from 'aws-amplify';
+import { getAmplifyOutputs } from '@utils/get-amplify-outputs';
+
+Amplify.configure(getAmplifyOutputs(), { ssr: true });
```

Comment:
Does this need to be here or is this something that is global to the whole app and might be better in one of the `resource` files?

Reply:
the resource files aren't a part of the web app. this is code that every page has to run through as NHSNotifyAuthenticator is called from the layout

Reply:
Maybe the layout might be a better place for it.  It feels a bit more generic and not directly auth related.

Reply:
I'm thinking that we might need to add more config there in the future that is completely unrelated to this file.

Reply:
moved up to layout